src/CMakeLists.txt

```diff
@@ -132,14 +133,13 @@ if (WITH_C_API)
 endif ()
 
 # Install targets:
-SET(TARGETLIBEXPORT "dbcsr")
-install (TARGETS dbcsr EXPORT ${TARGETLIBEXPORT}
-         LIBRARY DESTINATION lib
-         ARCHIVE DESTINATION lib)
+install(TARGETS dbcsr EXPORT dbcsr
+  LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}")
 if (WITH_C_API)
-  install (TARGETS dbcsr_c EXPORT ${TARGETLIBEXPORT}
-          LIBRARY DESTINATION lib
-          ARCHIVE DESTINATION lib)
-  install (FILES "${CMAKE_SOURCE_DIR}/src/dbcsr.h"
-           DESTINATION include)
+  install(TARGETS dbcsr_c EXPORT dbcsr_c
+    LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+    ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+  install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/dbcsr.h"
```

Comment:
it's NOT `CMAKE_CURRENT_SOURCE_DIR`, it should be `CMAKE_SOURCE_DIR`.

Reply:
oh, never mind, it looks like you moved the header recently... I tested that patch on release candidate.

Reply:
could you check out-of-source build? not that it should matter, but to be sure...

Reply:
I'm only building out-of-source and would consider in-source building unsupported ;-)
The relation between `CMAKE_CURRENT_SOURCE_DIR` and `CMAKE_SOURCE_DIR` for `src/` is: `CMAKE_CURRENT_SOURCE_DIR=CMAKE_SOURCE_DIR/src`

Reply:
> I'm only building out-of-source and would consider in-source building unsupported ;-)

I fully agree ;-)

> The relation between CMAKE_CURRENT_SOURCE_DIR and CMAKE_SOURCE_DIR for src/ is: CMAKE_CURRENT_SOURCE_DIR=CMAKE_SOURCE_DIR/src

I know that but I am still puzzled how/why `CMAKE_SOURCE_DIR` worked for me in the other PRs... But as I said, does not matter as long as you check that the header is being installed.